x-pack/legacy/plugins/reporting/server/lib/esqueue/__tests__/worker.js

```diff
@@ -380,12 +379,6 @@ describe('Worker class', function () {
         const { body } = getSearchParams(jobtype);
         expect(body).to.have.property('size', defaults.size);
       });
-
-      it('should observe the size option', function () {
-        const size = 25;
-        const { body } = getSearchParams(jobtype, { size });
-        expect(body).to.have.property('size', size);
-      });
```

Comment:
This is removed because the class is no longer responsible for tracking the size: the size just comes from a constant, and that is verified somewhere else in this test file